modules/api/src/main/java/org/apache/fluo/api/client/FluoAdmin.java

```diff
@@ -37,6 +37,7 @@
 
     private boolean clearZookeeper = false;
     private boolean clearTable = false;
+    private boolean updateWithoutInitializing = false;
```

Comment:
Not sure about the full implications of adding another field to InitializationOptions

Reply:
If `admin.updateSharedConfig()` fails when the application is running, then this field may not be needed.